server/game_websocket/room.ts

```diff
@@ -1,71 +1,52 @@
 // this is where the rooms are managed and created
 import { v4 as uuidv4 } from "uuid";
-type Rooms = { users: number; id: string; board: Board };
+type Rooms = {
```

Comment:
Current naming with rooms looks a bit confusing. Lets name it not Rooms, but`roomState` and then rename `roomState` property of `Room` class to `roomsState`.